diff --git a/bin/Makefile b/bin/Makefile
index 568c96c..396632c 100644
--- a/bin/Makefile
+++ b/bin/Makefile
@@ -99,7 +99,7 @@ CFLAGS = -O -DUNDERSCORE
 #FFLAGS = -g -O0 -C
 ##--------------------------
 ### Intel Fortran Compiler
-FC = ifort
+FC = gfortran
 FFLAGS = -O
 FFLOPT = -O
 #FTNLIB = -Vaxlib /usr/lib/C-ctype.o /usr/lib/C_name.o /usr/lib/ctype-info.o
@@ -108,12 +108,12 @@ FFLOPT = -O
 
 ##--------------------------
 ### Double precision option
-FFLAGS = -O -r8 -CB
-FFLOPT = -O -r8 -CB
-PLTOBJ = ../plotlib/libPltDP.a 
+FFLAGS = -O -fdefault-real-8 -B
+FFLOPT = -O -fdefault-real-8 -B
+PLTOBJ = ../plotlib/libPlt.a
 
-FFLAGS = -O -r8 -ftrapuv -fpe0
-FFLOPT = -O -r8 -ftrapuv -fpe0
+FFLAGS = -O -fdefault-real-8 -ftrapv -fdec
+FFLOPT = -O -fdefault-real-8 -ftrapv -fdec
 
 
 all:	 $(PROGS)
diff --git a/orrs/bin/Makefile_DP b/orrs/bin/Makefile_DP
index 782c858..5cf578b 100644
--- a/orrs/bin/Makefile_DP
+++ b/orrs/bin/Makefile_DP
@@ -11,8 +11,8 @@ FTNLIB =
 
 #========================================
 #  Uncomment for Intel Fortran Compiler
-FC = ifort
-FLG = -O -r8
+FC = gfortran
+FLG = -O -fdefault-real-8
 PLTLIB = -L/usr/X11R6/lib -lX11
 FTNLIB =
 
diff --git a/orrs/src/osmap.f b/orrs/src/osmap.f
index db14817..8ac21be 100755
--- a/orrs/src/osmap.f
+++ b/orrs/src/osmap.f
@@ -98,7 +98,7 @@ C
 C
 C---- set OSFILE to match the absolute OS database filename
       CHARACTER*128 OSFILE
-      DATA OSFILE / '/var/local/codes/orrs/osmapDP.dat' /
+      DATA OSFILE / '/Xfoil/orrs/osmapDP.dat' /
 ccc      DATA OSFILE / '/var/local/codes/orrs/osmap.dat' /
 c
 c      DATA OSFILE
diff --git a/plotlib/Makefile b/plotlib/Makefile
index 76bb09f..9ca8f17 100644
--- a/plotlib/Makefile
+++ b/plotlib/Makefile
@@ -69,12 +69,12 @@ OBJ3D  = plt_3D.o
 
 ###================================================
 ###  Default compilers and flags, install commands
-FC = f77
+FC = gfortran
 CC  = cc
 # Some fortrans need trailing underscores in C interface symbols (see Xwin.c)
 DEFINE = -DUNDERSCORE
 # Uncomment DP to make double-precision version
-#DP = -r8
+DP = -fdefault-real-8
 FFLAGS  = -O $(DP)
 CFLAGS  = -O $(DEFINE)
 AR = ar r 
diff --git a/plotlib/config.make b/plotlib/config.make
index 0870844..20c3081 100644
--- a/plotlib/config.make
+++ b/plotlib/config.make
@@ -51,11 +51,11 @@ DEFINE = -DUNDERSCORE
 
 ###-------------------------------------------------------------------------
 ###  Uncomment for Linux, using Intel Fortran compiler 8.x
-FC = ifort
+FC = gfortran
 CC  = gcc
  
 # Uncomment to make double-precision version
-#DP = -r8
+DP = -fdefault-real-8
 
 FFLAGS  = -O3 $(DP)
 CFLAGS  = -O3 $(DEFINE)
diff --git a/src/pplot.f b/src/pplot.f
index 249c4fd..024726b 100644
--- a/src/pplot.f
+++ b/src/pplot.f
@@ -54,7 +54,7 @@ C...Try to read default file "pplot.def" for stored plot setup
       OPEN(LU,FILE='pplot.def',STATUS='OLD',ERR=2)
       CALL RDDEF(LU,LERR)
       CLOSE(LU)
-      IF(LERR) THEN
+      IF(LERR.ne.0) THEN
        WRITE(*,*)
        WRITE(*,*) 'Read error on file  pplot.def'
        WRITE(*,*) 'Using default settings'
